buildsystem/cpp.cmake

```diff
@@ -88,7 +89,7 @@ function(finalize_binary target_name output_name type)
 				message(FATAL_ERROR "finalize_binary flag 'allow_no_undefined' is only valid for libraries!")
 			endif()
 
-			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize")
+			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize" AND NOT MSVC)
```

Comment:
Undefined symbols are only discovered at link-time for executables, not for libraries like `libopenage`. Those cflags ensure that errors are shown nevertheless. I guess msvc has something that does the same.